sdk/ai/Azure.AI.Projects/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ### Features Added
 
+* Added objects to inform Agents about Azure Functions.
+* Added `OpenApiTool` for Agents, which creates and executes a REST function defined by an OpenAPI spec.
+* Add `parallel_tool_calls` parameter to allow parallel tool execution for Agents.
+
 ### Breaking Changes
 
 ### Bugs Fixed
```

Comment:
Could you update the changelog to include an entry for the bug fix related to [issue #47244](https://github.com/Azure/azure-sdk-for-net/issues/47244) that you resolved [here](https://github.com/Azure/azure-sdk-for-net/pull/47272)?

Reply:
Fixed.